src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Random random = new Random();
```

Comment:
Don't create several instances of the same class if you can use only one instance for your purposes. In this case, you can create a single instance of `Random` as a class-level field instead of creating a new instance each time the method is called.